src/main/java/org/jenkinsci/plugins/durabletask/BourneShellScript.java

```diff
@@ -231,7 +235,7 @@ private FilePath pidFile(FilePath ws) throws IOException, InterruptedException {
                         if (pidFile.exists()) {
                             listener.getLogger().println("still have " + pidFile + " so heartbeat checks unreliable; process may or may not be alive");
                         } else {
-                            listener.getLogger().println("wrapper script does not seem to be touching the log file in " + controlDir);
+                            listener.getLogger().println("s " + controlDir);
```

Comment:
Uh…?

Reply:
Erm, I have no idea where that came from.  Best guess It was a hotkeying error somewhere, but fixing now.